### ISSUE TITLE
Add tests for recent commits

### ISSUE THREAD

**Issue by dstansby** [MAVENSDC/cdflib#232]
👋 @bryan-harter, it would be good to add some tests to the test suite to make sure the fixes you recently made in versions 1.1.1 and 1.1.2 stay fixed in the future.

**Comment by bryan-harter** [comment]
Yes sorry I've kinda been rushing the commits, we're using cdflib to write the CDF files for the IMAP mission (specifically, xarray_to_cdf), so I wanted to get things fixed quickly so the different instruments could get started on CDF generation!
